src/root-page-manager.ts

```diff
@@ -73,6 +74,9 @@ export class RootPageManager implements Initializable {
       metadata.author = card.by.name;
       metadata.publishedTime = new Date(card.postedAt).toISOString();
     }
+    if (card && card.searchText) {
+      searchText = "<p>" + card.searchText + "</p>";
```

Comment:
I believe that searchText is already cleaned and text-only, but it seems like it would be safer for you to escape it. No?  escapeHtml() is already used elsewhere in this file.

Reply:
Just realized that another way to do this is using Mustache.  You can get it to escape or not (based on whether you include & before the name, I think).  But then the <p></p> should be inside the template, rather than here.

Reply:
good catch. Will fix it